ECMAScript/react/tsconfig.json

```diff
@@ -1,9 +1,7 @@
 {
   "extends": "../tsconfig.base.json",
   "compilerOptions": {
-    "paths": {
-      "@/*": ["./src/*"]
```

Comment:
What's wrong with the `paths` option here? Are the imports not being resolved properly after transpilation?

Reply:
No this was working fine. It's more about simplifying the configuration as much as possible. We get no real value from having the aliases in this project.